zebra-network/src/peer_set/initialize.rs

```diff
@@ -125,21 +119,18 @@ where
         let _ = demand_tx.try_send(());
     }
 
-    tokio::spawn(
-        crawl_and_dial(
-            config.new_peer_interval,
-            demand_tx,
-            demand_rx,
-            candidates,
-            connector,
-            peerset_tx,
-        )
-        .map(|result| {
-            if let Err(e) = result {
-                error!(%e);
-            }
-        }),
-    );
+    let crawl_guard = tokio::spawn(crawl_and_dial(
```

Comment:
Just a q: is naming the `JoinHandle`s returned by `tokio::spawn` 'guards'  a convention?

Reply:
Yeah, particularly in C++ "guard" is commonly used to indicate that some object is managing resources with [RAII](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization).

Reply:
TIL RAII! 🤓